server/datastore/mysql/apple_mdm_test.go

```diff
@@ -3253,7 +3235,7 @@ func testMDMAppleBootstrapPackageCRUD(t *testing.T, ds *Datastore) {
 func testListMDMAppleCommands(t *testing.T, ds *Datastore) {
 	ctx := context.Background()
 
-	createRawCmd := func(reqType, cmdUUID string) string {
+	createRawAppleCmd := func(reqType, cmdUUID string) string {
```

Comment:
Wondering if the top-level `createRawAppleCmd` should not accept a `reqType` too and then always use this instead?